backend/src/main/java/com/selflearn/backend/security/dtos/LoginDto.java

```diff
@@ -0,0 +1,19 @@
+package com.selflearn.backend.security.dtos;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+
+@Getter
+public class LoginDto {
```

Comment:
consider using Record instead of Class for immutable class
https://www.baeldung.com/java-record-keyword